spark_auto_mapper/automappers/with_column_base.py

```diff
@@ -62,13 +62,26 @@ def get_column_spec(self, source_df: Optional[DataFrame]) -> Column:
                 source_df=source_df, current_column=None, parent_columns=None
             )
             if self.skip_if_columns_null_or_empty:
-                columns_to_check = f"b.{self.skip_if_columns_null_or_empty[0]}"  # TODO: handle more than one
-                # wrap column spec in when
-                column_spec = when(
-                    col(columns_to_check).isNull()
-                    | col(columns_to_check).eqNullSafe(""),
-                    lit(None),
-                ).otherwise(
+                is_first_when_case = True
+                for columns in self.skip_if_columns_null_or_empty:
+                    column_to_check = f"b.{columns}"
+                    # wrap column spec in when
+                    column_spec = (
+                        when(
+                            col(column_to_check).isNull()
+                            | col(column_to_check).eqNullSafe(""),
+                            lit(None),
+                        )
+                        if is_first_when_case
+                        else column_spec.when(
+                            col(column_to_check).isNull()
+                            | col(column_to_check).eqNullSafe(""),
+                            lit(None),
+                        )
+                    )
+                    is_first_when_case = False
```

Comment:
can be simplified. Instead of maintaining is_first_when_case, just make use of `if column_spec` condition

Reply:
we cannot use column_spec here as we've initialized column_spec = lit(None). So, if we'll make the if column_spec is lit(None), then that could also be a case where row value is coming out to None. That is the reason why flag has been used.